code/modules/mob/living/basic/farm_animals/deer.dm

```diff
@@ -0,0 +1,52 @@
+/mob/living/basic/deer
+	name = "doe"
+	desc = "A gentle, peaceful forest animal. How did this get into space?"
+	icon_state = "deer-doe"
+	icon_living = "deer-doe"
+	icon_dead = "deer-doe-dead"
+	gender = FEMALE
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	speak_emote = list("grunts","grunts lowly")
+	speed = 0.8 //deer are fuckin fast when they want to be
+	see_in_dark = 6
+	faction = list("deer") //humans are neutral, so if the deer was neutral it wouldn't run from them.
+	butcher_results = list(/obj/item/food/meat/slab = 3)
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently nudges"
+	response_disarm_simple = "gently nudges aside"
+	response_harm_continuous = "kicks"
+	response_harm_simple = "kick"
+	attack_verb_continuous = "bucks"
+	attack_verb_simple = "buck"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	health = 75
+	maxHealth = 75
+	blood_volume = BLOOD_VOLUME_NORMAL
+	footstep_type = FOOTSTEP_MOB_SHOE
+
+	ai_controller = /datum/ai_controller/basic_controller/deer
+
+/datum/ai_controller/basic_controller/deer
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic,
+	)
+
+	ai_traits = STOP_MOVING_WHEN_PULLED
+	ai_movement = /datum/ai_movement/basic_avoidance
+
+	//IMPLEMENT IDLE BEHAVIOR HERE
```

Comment:
requires https://github.com/tgstation/tgstation/pull/61455,

will allow for a generic idle datum